<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>   掌柜的</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string> 出售 </string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLCDNumber" name="outPrice">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(158, 255, 153)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QWidget" name="widget_5" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>104</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pBClear">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(216, 149, 55);</string>
             </property>
             <property name="text">
              <string>清零</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lEBarCode">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pBSell">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(0, 207, 0);</string>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTextEdit" name="tEReceipt">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.98, y1:0.96, x2:1, y2:0, stop:0.422886 rgba(130, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_6" native="true">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>最大金额9999.9元</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>50</pointsize>
              </font>
             </property>
             <property name="text">
              <string>元</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string> 入库 </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="text">
              <string>商品名</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lECommodityName"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>元</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4" rowspan="5">
            <widget class="QLabel" name="lPicture">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="text">
              <string>商品图片</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lEPriceIn"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lEBarCodeIn"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="text">
              <string>商品条形码</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="text">
              <string>售价</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lEPrice"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="text">
              <string>进价</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>元</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="text">
              <string>商品大类</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cBCommodityClass"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>251</width>
                <height>42</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string>现有库存量</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="sBCurrentCount">
              <property name="geometry">
               <rect>
                <x>280</x>
                <y>30</y>
                <width>165</width>
                <height>48</height>
               </rect>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>460</x>
                <y>30</y>
                <width>126</width>
                <height>42</height>
               </rect>
              </property>
              <property name="text">
               <string>  增加</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="sBAddCount">
              <property name="geometry">
               <rect>
                <x>600</x>
                <y>30</y>
                <width>165</width>
                <height>48</height>
               </rect>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>800</x>
                <y>30</y>
                <width>126</width>
                <height>42</height>
               </rect>
              </property>
              <property name="text">
               <string>  减少</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="sBMinusCount">
              <property name="geometry">
               <rect>
                <x>950</x>
                <y>30</y>
                <width>165</width>
                <height>48</height>
               </rect>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>累计出售总量</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lETotalSales"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>折扣</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lEDiscount"/>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="pBUpdate">
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(0, 191, 0)</string>
                </property>
                <property name="text">
                 <string>更新信息</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QPushButton" name="pBAddCommodity">
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255, 113, 12)</string>
                </property>
                <property name="text">
                 <string>新增商品</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <attribute name="title">
        <string> 仓库 </string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lESearch">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="pBDelete">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(232, 0, 0)</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pBAdd">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 190, 0)</string>
          </property>
          <property name="text">
           <string>添加/更新</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pBRefresh">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(253, 255, 134)</string>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="8">
         <widget class="QTableWidget" name="tWWarehouse">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>30</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(191, 255, 191);
font: 30pt &quot;Arial&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string> 订购 </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="tLWebView">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>36</pointsize>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.98, y1:0.96, x2:1, y2:0, stop:0.422886 rgba(130, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: italic 36pt &quot;Arial&quot;;
text-decoration: underline;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tLviewYC">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.98, y1:0.96, x2:1, y2:0, stop:0.422886 rgba(130, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: italic 36pt &quot;Arial&quot;;
text-decoration: underline;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionkfxx"/>
   </widget>
   <addaction name="menuinfo"/>
  </widget>
  <action name="actionkfxx">
   <property name="text">
    <string> 联系我们</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
